pykeybasebot/kvstore_client.py

```diff
@@ -0,0 +1,115 @@
+import json
+from typing import Any, Dict, Union
+
+from .errors import DeleteNonExistentError, RevisionError
+from .types import keybase1
+
+
+class KVStoreClient:
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def put(
+        self,
+        team: str,
+        namespace: str,
+        entryKey: str,
+        entryValue: str,
+        revision: Union[int, None] = None,
+    ) -> keybase1.KVPutResult:
+        await self.bot.ensure_initialized()
+        args: Dict[str, Any] = {
+            "method": "put",
+            "params": {
+                "options": {
+                    "team": team,
+                    "namespace": namespace,
+                    "entryKey": entryKey,
+                    "entryValue": entryValue,
+                }
+            },
+        }
+        if revision:
+            args["params"]["options"]["revision"] = revision
+        try:
+            res = await self.execute(args)
+            return keybase1.KVPutResult.from_dict(res)
+        except Exception as e:
+            if e.args[0]["code"] == 2760:
```

Comment:
suggest pulling 2760 up into a constant (just defined at the top of this file) that says what it is: e.g. `CLIENT_STATUS_STORAGE_WRONG_REVISION`?

Reply:
oh - or in the errors file. that might be better.